tox.ini

```diff
@@ -84,10 +84,9 @@ description = Test bundles
 commands =
     pytest -v --tb native --asyncio-mode=auto {[vars]test_path}/integration/test_bundle_deployment.py --keep-models --log-cli-level=INFO -s {posargs}
 setenv =
-    1.7: BUNDLE_PATH = "./releases/1.7/stable/kubeflow/bundle.yaml"
```

Comment:
There are other references of 1.7 in this file and in .github/dependencies.yaml and .github/workflows/ 
You don't have to remove them in this PR since it is concerned with the bundle tests, but just to keep it in mind

Reply:
Maybe I should have a separate PR for removing 1.7 everywhere

Reply:
[FYI](https://github.com/canonical/bundle-kubeflow/issues/1004).